src/main/java/com/puppycrawl/tools/checkstyle/checks/metrics/BooleanExpressionComplexityCheck.java

```diff
@@ -74,6 +74,23 @@
  * <pre>
  * &lt;module name="BooleanExpressionComplexity"/&gt;
  * </pre>
+ * <p>Code Example:</p>
+ * <pre>
+ * public class Test
+ * {
+ *  public static void main(String ... args)
+ *  {
+ *   boolean a = true;
+ *   boolean b = false;
+ *
+ *   boolean c = (a &amp; b) | (b &amp; a); // OK, &amp; and | are counted as 1 which gives
```

Comment:
`// OK, 1(&amp;) + 1(|) + 1(&amp;) = 3`

It is more compact and more descriptive.

Please apply to other examples.
Remember you can wrap expression on few lines to give more space for comments. Only comment ok/violation should be in the same line, explanation could be around if have no space for it, but ideally on the same line.

After update, do not forget to update image and CLI outputs.

Reply:
@romani done

Reply:
@romani Can you please review and let me know if any further changes are required.Thanks!